### ISSUE TITLE
vehicle position is freezing after position change

### ISSUE THREAD

**Issue by so-lar** [altmp/altv-issues#788]
`Build 2189, branch release`

**Current behavior**
If you change the position of a vehicle the vehicle is getting freezed. The position is getting unfreezed, if you restream the vehicle and if you are the only player around (so if you become the network owner).
Example video: [http://cloud.xyx.solar/s/2sWoZTPe5fJM5dS](http://cloud.xyx.solar/s/2sWoZTPe5fJM5dS)

**Expected behavior**
The vehicle should get unfreezed, so it doesn´t get stuck in air.

**Steps to reproduce**
Spawn a vehicle and change the position 

```
let veh = new alt.Vehicle('rhapsody', 1023.2440, -779.0901, 57.8740, 0, 0, 2.2758);                                     
veh.pos = player.pos;
```

**Comment by C0kkie** [comment]
Can you try if you can reproduce it on newest rc version?

**Comment by C0kkie** [comment]
Fixed on 7.0
